source/auth/auth.rst

```diff
@@ -622,7 +622,8 @@ The MongoDB SCRAM-SHA-256 mechanism works similarly to the SCRAM-SHA-1
 mechanism, with the following changes:
 
 - The SCRAM algorithm MUST use SHA-256 as the hash function instead of SHA-1.
-- User names MUST be prepared with SASLprep, per RFC 5802.
+- User names MUST NOT be prepared with SASLprep.  This intentionally
+  contravenes the "SHOULD" provision of RFC 5802.
```

Comment:
Let's explain the reason for this in the Q&A please.

Reply:
Will do.